.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
     unzip -o terraform_${TF_VERSION}_linux_amd64.zip -d ~/bin
     rm terraform_${TF_VERSION}_linux_amd64.zip
   - pip install --user yamllint
-  - eval "$(gimme 1.6)"
+  - eval "$(gimme 1.8)"
```

Comment:
Do you know what the errors were? Can you split it out to a separate commit with the details?